share/commands/agent-info

```diff
@@ -36,6 +36,14 @@ else
   ROLES="${ROLES}, rudder-agent"
 fi
 
+if [ -d /var/rudder/share ]
+then
+  ROLES="${ROLES}, rudder-relay"
+elif [ -d /var/rudder/inventories/incoming ]
+then
+  ROLES="${ROLES}, rudder-relay (not registered)"
```

Comment:
Currently, it looks to me as if this will display a "rudder-relay" role on root servers too. Surely we should protect against this?